system/security/plugins/jwtSecurity/CMakeLists.txt

```diff
@@ -24,7 +24,7 @@ option(BUILD_TESTS "Configure CMake to build tests (or not)" ON) # Copy from jwt
 set(BUILD_TESTS OFF)
 option(BUILD_EXAMPLES "Configure CMake to build examples (or not)" ON) # Copy from jwt-cpp/CMakeLists.txt
 set(BUILD_EXAMPLES OFF)
-add_subdirectory(jwt-cpp EXCLUDE_FROM_ALL)
+# add_subdirectory(jwt-cpp EXCLUDE_FROM_ALL)
```

Comment:
@richardkchapman I commented this out, but it looks like it could be deleted?

Reply:
@dcamper Do you know?

Reply:
https://cmake.org/cmake/help/latest/prop_tgt/EXCLUDE_FROM_ALL.html

IIRC this was part of trying to tightly control the caching/recompilation phase of the underlying code.  Perhaps too tightly.

I would suggest keeping the add_subdirectory() command but deleting the EXCLUDE_FROM_ALL option and see if the results are right.

Reply:
Changed as per Dans suggestion